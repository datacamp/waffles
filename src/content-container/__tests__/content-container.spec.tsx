import { render } from '@testing-library/react';

import { ContentContainer } from '../index';

describe('ContentContainer', () => {
  it('renders correctly', () => {
    const { getByTestId } = render(
      <ContentContainer data-testid="container">
        Unofficial Fanpage of Taylor Swift
      </ContentContainer>,
    );

    const contentContainer = getByTestId('container');

    expect(contentContainer).toMatchSnapshot();
  });
});
