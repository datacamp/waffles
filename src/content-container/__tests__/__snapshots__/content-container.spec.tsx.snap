// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentContainer renders correctly 1`] = `
.emotion-0 {
  padding: 16px;
  max-width: 1000px;
  overflow: hidden;
}

@media screen and (min-width: 820px) {
  .emotion-0 {
    padding: 32px;
  }
}

@media screen and (min-width: 1480px) {
  .emotion-0 {
    margin-left: auto;
    margin-right: auto;
    max-width: 1200px;
  }
}

<div
  class="emotion-0"
  data-testid="container"
>
  Unofficial Fanpage of Taylor Swift
</div>
`;

exports[`ContentContainer renders correctly when noSidebar flag is passed 1`] = `
.emotion-0 {
  padding: 16px;
  max-width: 1000px;
  overflow: hidden;
  margin-left: auto;
  margin-right: auto;
}

@media screen and (min-width: 820px) {
  .emotion-0 {
    padding: 32px;
  }
}

@media screen and (min-width: 1480px) {
  .emotion-0 {
    margin-left: auto;
    margin-right: auto;
    max-width: 1200px;
  }
}

<div
  class="emotion-0"
  role="main"
>
  Unofficial Fanpage of Taylor Swift
</div>
`;
